docker-compose.yml

```diff
@@ -22,14 +22,20 @@ services:
       - HTFEED_CONFIG=/usr/local/feed/etc/config_test.yml
       - FEED_HOME=/usr/local/feed
       - TEST=1
+      - CLAMAV_HOST=clamav
+      - CLAMAV_PORT=3310
 
   travis_test:
     build: .
+    volumes:
+      - .:/usr/local/feed
     environment:
       - HTFEED_CONFIG=/usr/local/feed/etc/config_test.yml
       - FEED_HOME=/usr/local/feed
       - TEST=1
-    command: bin/wait-for --timeout=300 mariadb:3306 -- make test TEST_VERBOSE=1
+      - CLAMAV_HOST=clamav
+      - CLAMAV_PORT=3310
+    command: bin/wait-for --timeout=300 mariadb:3306 -- bin/wait-for --timeout=300 clamav:3310 -- make test TEST_VERBOSE=1
```

Comment:
ah, this is probably not quite working -- I can probably adjust wait-for to take multiple things it can wait for, if it can't already

Reply:
I'll look at the wait-for source. I expected it to need a tweak or two but then it seemed to work.

Reply:
Already fixed, commit incoming (just need to fix something with the config that's causing the tests to fail)

Reply:
I messed around with using `$self` in the PREMIS config as indicated in the comments (etc/sample_config/00_base_config.yml and etc/config/00_base_config.yml), so I could get the ClamAV version from the daemon directly, but could not get that to work.